### PR TITLE
fix(user): hide recently played events meta [V5.2]

### PR DESCRIPTION
**Pull request by wescopeland**
Prevents experiences like this from occurring:

image

If the game has the events console ID, the "Last played..." and "Mastered in..." labels will be hidden.

### CODE REVIEW

File: resources/views/platform/components/game-list-item/primary-meta.blade.php [hunk 1]
```diff
@@ -101,7 +102,7 @@
     @endif
 
     {{-- c.progress-pmeta__root > div --}}
-    <div @if ($variant === 'user-recently-played') class="flex !flex-col-reverse" @endif>
+    <div @if ($variant === 'user-recently-played' && $consoleId != 101) class="flex !flex-col-reverse" @endif>
```
[the remark this answers is not in the document]
Comment (wescopeland) [reply resources/views/platform/components/game-list-item/primary-meta.blade.php]: Whoops... going a little too fast. Resolved in latest.

Screenshot 2023-11-16 at 5 20 06 PM

**Review by Jamiras — APPROVED** [review]
I'd argue that events shouldn't appear in the Recently **Played** Games at all, but this is a good start.

**Comment by wescopeland** [comment]
> I'd argue that events shouldn't appear in the Recently Played Games at all, but this is a good start.

I like this idea much more than my current implementation, as it reduces the complexity of the game list item components.

I've rewritten this PR's implementation to simply exclude event games from the recently played list.

**Review by Jamiras — APPROVED** [review]
Minor: when an event game is removed from the recent list, it's not replaced with a non-event game, so you end up with Last 4 Games Played instead of Last 5 Games Played.

We can address that some other time.

**Pull request merged**